thirdeye-ui/src/app/components/subscription-group-wizard/subscription-group-wizard.component.tsx

```diff
@@ -12,214 +12,162 @@
  * See the License for the specific language governing permissions and limitations under
  * the License.
  */
-import { Box, Button, Grid, Typography } from "@material-ui/core";
-import React, { FunctionComponent, useEffect, useState } from "react";
+
+import { Box, Button, Grid } from "@material-ui/core";
+import React, { FunctionComponent, useEffect, useMemo, useState } from "react";
 import { useTranslation } from "react-i18next";
+import { useParams, useSearchParams } from "react-router-dom";
 import {
-    DataGridSelectionModelV1,
-    DataGridV1,
     PageContentsCardV1,
     PageContentsGridV1,
 } from "../../platform/components";
-import { Alert } from "../../rest/dto/alert.interfaces";
 import {
-    EmailScheme,
-    NotificationSpec,
+    AlertAssociation,
     SubscriptionGroup,
 } from "../../rest/dto/subscription-group.interfaces";
-import { getUiSubscriptionGroup } from "../../utils/subscription-groups/subscription-groups.util";
-import { GroupsEditor } from "./groups-editor/groups-editor.component";
-import { SubscriptionGroupPropertiesForm } from "./subscription-group-properties-form/subscription-group-properties-form.component";
 import {
-    generateDefaultSelection,
+    getConfigurationPath,
+    getSubscriptionGroupsAllPath,
+    getSubscriptionGroupsCreatePath,
+    getSubscriptionGroupsUpdatePath,
+} from "../../utils/routes/routes.util";
+import { PageHeader } from "../page-header/page-header.component";
+import { PageHeaderProps } from "../page-header/page-header.interfaces";
+import { AlertsDimensions } from "./alerts-dimensions/alerts-dimensions.component";
+import { SubscriptionGroupDetails } from "./subscription-group-details/subscription-group-details.component";
+import {
+    Association,
+    SubscriptionGroupsWizardParams,
+    SubscriptionGroupViewTabs,
+    SubscriptionGroupWizardProps,
+} from "./subscription-group-wizard.interface";
+import {
+    getAssociations,
+    SelectedTab,
     validateSubscriptionGroup,
-} from "./subscription-group-whizard.utils";
-import { SubscriptionGroupWizardProps } from "./subscription-group-wizard.interfaces";
+} from "./subscription-group-wizard.utils";
 
 export const SubscriptionGroupWizard: FunctionComponent<SubscriptionGroupWizardProps> =
-    ({ subscriptionGroup, alerts, onCancel, submitBtnLabel, onFinish }) => {
+    ({
+        alerts,
+        submitBtnLabel,
+        subscriptionGroup,
+        enumerationItems,
+        onCancel,
+        onFinish,
+        isExisting = false,
```

Comment:
The parent page should pass in what to use for the label for the submit button and anything specific to create or update

Reply:
Removed `isExisting`. The wizard will now be unaware of all such distinctions. The primary reason to have this prop was a common PageHeader, but that has already been resolved so this has be removed.